filters/include/pcl/filters/impl/uniform_sampling.hpp

```diff
@@ -43,19 +43,13 @@
 
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 template <typename PointT> void
-pcl::UniformSampling<PointT>::applyFilter (PointCloud &output)
+pcl::UniformSampling<PointT>::applyFilter (Indices &indices)
```

Comment:
I think we need to keep the old method still but add deprecation warning for removal in 1.17.

Reply:
This function is called through pcl::FilterIndices<PointT>::applyFilter (PointCloud &output), so it should not conflict with users' old code.

Reply:
`FilterIndices` has [that method](https://pointclouds.org/documentation/classpcl_1_1_filter_indices.html#a237872d14a14ba3881592d3106cf4338) which should take its place, right?

Reply:
Ahh, yeah ofc.